src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,145 @@
+package subway;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.annotation.DirtiesContext;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static subway.extracts.LineExtracts.*;
+import static subway.extracts.ResponseExtracts.상태코드;
+import static subway.extracts.StationExtracts.지하철역_생성_응답_ID_추출;
+import static subway.requests.LineRequests.*;
+import static subway.requests.StationRequests.지하철역_생성_요청하기;
+
+@DisplayName("지하철노선 관련 기능")
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class LineAcceptanceTest {
+
+    @LocalServerPort
+    private int port;
+
+    @BeforeEach
+    void setUp() {
+        RestAssured.port = port;
+    }
+
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철노선을 생성한다.")
+    @Test
+    void createLine() {
+        String 강남역_ID = 지하철역_생성_응답_ID_추출(지하철역_생성_요청하기("강남역"));
+        String 양재역_ID = 지하철역_생성_응답_ID_추출(지하철역_생성_요청하기("양재역"));
+        String lineName = "신분당선";
+
+        // when
+        ExtractableResponse<Response> 지하철노선_생성_응답 = 지하철노선_생성_요청하기(lineName, 강남역_ID, 양재역_ID);
+
+        // then
+        assertThat(상태코드(지하철노선_생성_응답)).isEqualTo(HttpStatus.CREATED);
+        assertThat(지하철노선_목록_조회_응답_노선_이름_추출(지하철노선_목록_조회_요청하기())).contains(lineName);
```

Comment:
```suggestion
        지하철노선_목록_조회_시_노선_조회_가능(강남역_ID, 양재역_ID)
```

이렇게 검증 자체를 메서드로 분리해보면
시나리오 요구사항의 가독성은 높아지지만 테스트의 구체적인 행동은 감춰지네요.
어떤 것을 더 중점적으로 생각하는게 좋을지 고민해보시면 좋을 것 같아요 :)